src/index.js

```diff
@@ -1 +1,55 @@
 import './style.css';
+// import {GetScores, MakeGame, addScore} from './APIdata';
+import APIdata from './APIdata.js';
+
+const Scores = document.querySelector('.scores-list');
+const Form = document.querySelector('.fomr');
+const refreshBtn = document.querySelector('#refresh-btn');
+let gameIndex;
+
+const clearElement = (element) => {
+  while (element.firstChild) {
+    element.removeChild(element.firstChild);
+  }
+};
+
+const render = async (gameIndex) => {
+  const data = await APIdata.GetScores(gameIndex);
+  clearElement(Scores);
+  data.result.forEach((element) => {
+    const li = document.createElement('li');
+    const span1 = document.createElement('span');
+    span1.classList.add('name-span');
+
+    const span2 = document.createElement('span');
+    span2.classList.add('score-span');
+
+    span1.innerHTML = element.user;
+    span2.innerHTML = element.score;
+
+    li.appendChild(span1);
+    li.appendChild(span2);
+    Scores.appendChild(li);
+  });
+};
+
+const Storage = localStorage.getItem('Game Index');
+if (!Storage) {
+  APIdata.MakeGame().then(() => {
+    gameIndex = localStorage.getItem('Game Index');
+  });
+} else {
+  gameIndex = localStorage.getItem('Game Index');
+  render(gameIndex);
+}
+
+Form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  if (Form.name.value === '' || Form.score.value === null) return;
+  APIdata.addScore(gameIndex, Form.name.value, Form.score.value);
+  render(gameIndex);
+});
+
+refreshBtn.addEventListener('click', () => {
+  render(gameIndex);
+});
```

Comment:
- [ ] Nice job with the use of arrow function syntax in all your function expressions. However, I added my name and score to the leader board game and click on the refresh button but my score was not displayed on the page. Please try to fix this bug so you can have a working app.

Reply:
I am using a proxy server to avoid having cors errors, the app works, just might give it time